src/scripts/Common/Base.cpp

```diff
@@ -751,6 +798,11 @@ void MoonScriptCreatureAI::Emote(const char* pText, TextType pType, uint32 pSoun
     if (pSoundId > 0) _unit->PlaySoundToSet(pSoundId);
 }
 
+void MoonScriptCreatureAI::Emote(uint32 scripttext)
```

Comment:
> do NOT use old typedefs such as uint32, uint8 etc - use the new fixed-size types such as uint32_t and uint8_t for all new code
http://board.ascemu.org/index.php/Thread/196-Welcome-to-2017-and-with-it-modern-C/